Makefile

```diff
@@ -235,7 +235,19 @@ parse-test-e2e-operator-log:
 test-unit:
 	$(info Running unit test: $@)
 	$(Q)GO111MODULE=$(GO111MODULE) GOCACHE=$(GOCACHE) \
-		go test $(shell GOCACHE="$(GOCACHE)" go list ./...|grep -v e2e) -v -mod vendor $(TEST_EXTRA_ARGS)
+		go test $(shell GOCACHE="$(GOCACHE)" go list ./...|grep -v e2e|grep -v examples) -v -mod vendor $(TEST_EXTRA_ARGS)
+
+.PHONY: test-examples
+## Runs the examples tests without code coverage
+test-examples:
```

Comment:
running examples is actually equivalent to running acceptance tests, and this PR is supposed to set the basis for that, we would also need to change CI afterthat. Thus, how about to call this rule `test-acceptance`?

Reply:
Agreed